styles/components/avatar.css

```diff
@@ -0,0 +1,40 @@
+.avatar {
+  position: relative;
+  margin: 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.avatar__img--rounded {
+  object-fit: cover;
+  border-radius: 50%;
+}
+.avatar__img--squared {
+  object-fit: cover;
+  border-radius: 12.5%;
+}
+.avatar__badge {
```

Comment:
Seems for your larger images the badges are going outside the image (only for rounded avatars)
would suggest that you look into creating different badge classes for different avatar sizes, so that badges sit along the rounded edge of avatars

Reply:
Yes, great suggestions and also started making a lot of sense. so instead of using different classes for each size I have started using percentage values and overcame that issue. Now the badges are looking proper.

![image](https://user-images.githubusercontent.com/32711472/153038297-e2265bb0-ce65-4741-8bff-d2d07809f591.png)

Thanks a lot for the review! 🎉